src/libpmemfile-posix/dir.c

```diff
@@ -357,10 +357,9 @@ vinode_update_parent(PMEMfilepool *pfp,
  * Note: caller must hold WRITE lock on parent.
  * Must be called in a transaction.
  */
-struct pmemfile_vinode *
+TOID(struct pmemfile_inode)
 vinode_new_dir(PMEMfilepool *pfp, struct pmemfile_vinode *parent,
```

Comment:
"posix: remove inode_ref_new ... This means that sometimes we have to pass around inodes instead
of vinodes."
So shouldn't this function be called `inode_new_dir` now?